cmd/echo.go

```diff
@@ -125,18 +120,9 @@ func init() {
 					}
 					req := &pb.EchoRequest{
 						Response: &pb.EchoRequest_Content{Content: input}}
-					err := stream.Send(req)
-					if err != nil {
-						errLog.Fatalf("%+v", err)
-					}
-					stdLog.Printf("Sent Request: %s", proto.MarshalTextString(req))
-				}
-
-				resp, err := stream.CloseAndRecv()
-				if err != nil {
-					errLog.Fatalf("%+v", err)
+					stream.Send(req)
```

Comment:
> Why are we dropping error handling? Is it because we expect the interceptor to log it? 

Correct. The interceptor should simply log any type of response. I added some comments in code to explicitly state that the interceptors will handle responses.

> But then why are we checking in other places?
User error. We need some error checking to end streams and I forgot to remove the error checks.